### PR TITLE
Add content visibility option

### PR DESCRIPTION
**Pull request by artem-malko**
Add [content-visibility](https://web.dev/content-visibility) property to patches, cause it is not in MDN yet.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1492,6 +1492,18 @@ export interface StandardLonghandProperties<TLength = (string & {}) | 0, TTime =
...
+  /**
+   * **Syntax**: `visible | auto | hidden`
+   *
+   * **Initial value**: `visible`
+   *
+   * | Chrome | Firefox | Safari |  Edge  | IE  |
+   * | :----: | :-----: | :----: | :----: | :-: |
+   * | **85** |   No    |   No   | **85** | No  |
+   *
+   * @see https://developer.mozilla.org/docs/Web/CSS/content-visibility
```
Comment (artem-malko) [inline index.d.ts:1504]: Oh, I haven't seen that) So, should I just rebuild typings without my patch?)

**Comment by frenic** [comment]
Thank you! I added this PR https://github.com/mdn/data/pull/450 in hopes it gets merged before applying this patch

**Comment by artem-malko** [comment]
@frenic thx, so, lets wait for your pr)

**Comment by frenic** [comment]
https://github.com/mdn/data/pull/450 is merged

**Pull request closed without merging**